src/main/resources/io/jenkins/plugins/pipelinegraphview/consoleview/PipelineConsoleViewAction/index.jelly

```diff
@@ -8,6 +8,18 @@
             <j:out value="${h.generateConsoleAnnotationScriptAndStylesheet()}"/>
 
             <j:set var="controls">
+                <j:if test="${it.building}">
+                    <l:hasPermission permission="${it.cancelPermission}">
+                        <j:set var="proxyId" value="${h.generateId()}" />
+                        <st:bind value="${it}" var="cancelAction${proxyId}"/>
+                        <button id="pgv-cancel" data-success-message="${%Build cancelled}"
```

Comment:
This should have a dialog, like in Jenkins core, so people don't accidentally cancel a pipeline